### ISSUE TITLE
GitPages

### ISSUE THREAD

**Issue by miketweaver** [Ch3LL/CTF#2]
I think these resources would be better suited in something like a GitHub Page.

https://pages.github.com/

Thoughts?

**Comment by Gr3yR0n1n** [comment]
Sounds like a great idea. Ch3LL?

**Comment by Ch3LL** [comment]
yeah totally. I will get that setup when i get a minute of free time.
